core/message_pool_test.go

```diff
@@ -226,3 +226,11 @@ func mustResignMessage(signer types.Signer, message *types.SignedMessage, f func
 func signMessage(signer types.Signer, message types.Message) (*types.SignedMessage, error) {
 	return types.NewSignedMessage(message, signer, types.NewGasPrice(0), types.NewGasUnits(0))
 }
+
+func mustAdd(t *testing.T, p *core.MessagePool, height uint64, msgs ...*types.SignedMessage) {
```

Comment:
rename to requireAdd per convention now that it doesn't panic

Reply:
It collides with the same name in `inbox_test.go`, but renamed to `reqAdd`.